src/olympia/activity/migrations/0019_merge_0018_auto_20221122_1313_0018_auto_20221128_1706.py

```diff
@@ -0,0 +1,14 @@
+# Generated by Django 3.2.16 on 2022-12-05 11:46
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('activity', '0018_auto_20221122_1313'),
+        ('activity', '0018_auto_20221128_1706'),
```

Comment:
could have just renamed the migration to 0019?

Reply:
I was too lazy to check which migrations I had already applied or not locally and revert them to get to the proper state, so I figured I'd use django migration's merge feature, that's why it exists after all!